uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/components/my-leave-calendar_test.js

```diff
@@ -67,6 +68,7 @@
 
           return $q.resolve(data);
         });
+        spyOn(AbsenceType, 'all').and.callThrough();
```

Comment:
True, but as `AbsenceType.all`gets called while init. I had to put it before component compilation